src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PrisonerSearchPrisoner.kt

```diff
@@ -18,6 +19,12 @@ data class PrisonerSearchPrisoner(
   @JsonFormat(pattern = "yyyy-MM-dd")
   val homeDetentionCurfewEligibilityDate: LocalDate? = null,
 
+  @JsonFormat(pattern = "yyyy-MM-dd")
+  var homeDetentionCurfewActualDate: LocalDate? = null,
+
+  @JsonFormat(pattern = "yyyy-MM-dd")
+  var homeDetentionCurfewEndDate: LocalDate? = null,
```

Comment:
had to expose all these field to this representation we get from prisoner search so I could copy on the representation we return to the frontend